src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let firstCopyState = { ...state };
```

Comment:
```suggestion
  let stateCopy = { ...state };
```
you will not have second)